src/js_parser.zig

```diff
@@ -23189,3 +23998,18 @@ const CommonJSWrapper = union(enum) {
     bun_dev: Expr,
     bun_js: void,
 };
+
+/// Equivalent of esbuild's js_ast_helpers.ToInt32
+fn floatToInt32(f: f64) i32 {
+    // Special-case non-finite numbers
+    if (!std.math.isFinite(f))
+        return 0;
+
+    const uint: u32 = @intFromFloat(@mod(@abs(f), 4294967296));
+    const int: i32 = @bitCast(uint);
+    return if (f < 0) @as(i32, 0) -% int else int;
```

Comment:
if `int` is max and `f` is negative, go will actually return `min(i32) - 1` instead of wrapping

Reply:
resolved